src/Orchard.Cms.Web/Modules/Orchard.OpenId/Controllers/AdminController.cs

```diff
@@ -207,7 +224,7 @@ IOpenIdService openIdService
                 if (model.Type == ClientType.Confidential)
                     roleNames = model.RoleEntries.Where(r => r.Selected).Select(r => r.Name).ToList();
 
-                var openIdApp = new OpenIdApplication { DisplayName = model.DisplayName, RedirectUri = model.RedirectUri, LogoutRedirectUri = model.LogoutRedirectUri, ClientId = model.ClientId, ClientSecret = Crypto.HashPassword(model.ClientSecret), Type = model.Type, SkipConsent = model.SkipConsent, RoleNames = roleNames };
+                var openIdApp = new OpenIdApplication { DisplayName = model.DisplayName, RedirectUri = model.RedirectUri, LogoutRedirectUri = model.LogoutRedirectUri, ClientId = model.ClientId, ClientSecret = Crypto.HashPassword(model.ClientSecret), Type = model.Type, SkipConsent = model.SkipConsent, RoleNames = roleNames, AllowAuthorizationCodeFlow = model.AllowAuthorizationCodeFlow, AllowClientCredentialsFlow = model.AllowClientCredentialsFlow, AllowImplicitFlow = model.AllowImplicitFlow, AllowPasswordFlow = model.AllowPasswordFlow, AllowRefreshTokenFlow = model.AllowRefreshTokenFlow, AllowHybridFlow = model.AllowHybridFlow };
```

Comment:
This line looks really long. Have you consider adding a few linebreaks? :trollface: